src/GraphQLParser.Tests/ParserTests.cs

```diff
@@ -171,13 +171,13 @@ public void Comments_on_NamedTypes_Should_Read_Correctly(IgnoreOptions options)
             frag.TypeCondition.Type.Comment.Text.ShouldBe("comment for named type from TypeCondition");
 
             var def2 = document.Definitions[1] as GraphQLObjectTypeDefinition;
-            def2.Interfaces[0].Comment.Text.ShouldBe("comment for named type from ImplementsInterfaces");
+            def2.Interfaces.Items[0].Comment.Text.ShouldBe("comment for named type from ImplementsInterfaces");
```

Comment:
Is it necessary to change everything like this?  Can we not add an `Item[int index]` property to these classes for convenience?  Can they implement `IList`?

Do these additional 'list' classes hold information that they were unable to previously?

Reply:
Looks like they hold can location information...

Reply:
> Do these additional 'list' classes hold information that they were unable to previously?

Yes. The existence of these classes allows you to write virtual methods passing them in.

>  Can we not add an Item[int index] property to these classes for convenience? Can they implement IList?

OK. I thought to do it, but then I forgot. They all really encapsulate only one collection of one type so it can be done.

Reply:
I'm okay either way.  I would implement `IReadOnlyList<T>` at a minimum -- basically `IEnumerable`, `Item` and `Count` -- so that we can iterate over the collection or use Item and Count for allocation-free iteration.

Reply:
But whatever you think.

Reply:
Done. Now I see only `VariableDefinitions`->`Variables` rename in tests.

Reply:
Other properties in definitions classes do not have _Definitions_ suffix, so I deleted it here.